### PR TITLE
[DIRAPI-353] automatic module names are added

### PR DESCRIPTION
**Pull request by solomax**
I was unable to test this due to missing release of `apacheds-test-framework`

### CODE REVIEW

**Comment by elecharny** [comment]
Patch applied. Thanks !

**Comment by solomax** [comment]
Good to know :)
naming can be changed after testing with `real` java11 project containing `module-info`

File: all/pom.xml [hunk 1]
```diff
@@ -126,6 +126,13 @@
               </artifactSet>
               <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
               <createSourcesJar>true</createSourcesJar>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <manifestEntries>
+                    <Automatic-Module-Name>apache.directory.ldap.api.all</Automatic-Module-Name>
```
Comment (seelmann) [inline all/pom.xml]: @solomax @elecharny should't the name here (and in other poms) start with `org.` prefix?

**Comment by elecharny** [comment]
Quite likely, I think.

**Comment by elecharny** [comment]
I just added the 'org.'  and committed the code. Thanks Stefan !

File: all/pom.xml (hunk 1 above)
Comment (solomax) [reply all/pom.xml]: I believe this is up to project maintainers :)))

**Pull request closed without merging**